site/smart-accounts/accounts/light-account.md

```diff
@@ -12,41 +12,49 @@ head:
       content: Follow this guide to use Light Account with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
   - - meta
     - name: twitter:title
-      content: Light Account Smart Contract
+      content: Light Account
   - - meta
     - name: twitter:description
       content: Follow this guide to use Light Account with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
 ---
 
-# Light Account Smart Contract
+# Light Account
 
-Light Account is a secure, gas-optimized, [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart account implementation.
+**Secure, optimized, and extendable**
 
-We started with the Ethereum Foundation’s canonical [SimpleAccount](https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/samples/SimpleAccount.sol) and added key improvements for production app developers:
```

Comment:
i'd also keep this?

Reply:
I feel like people don't really care that the light account is an improvement to SimpleAccount. What's important is that it is efficient, secure, and compatible/portable.

Even web3 devs, most of them don't know about the simple account reference impl by Ethereum, and even those that do know that it is a ref impl, not for production.

Reply:
sgtm

Reply:
good point, I see both sides here.  I lean toward keeping it for the next ~ 2 months because
* many current AA devs know about SimpleAccount and that is their point of reference
* this builds credibility for us -- we're based on the EF's official version and we improved upon it

i think by early next year this becomes less relevant to the audience and we've built up our brand as a contract developer, then we can remove this